packages/react-ui/components/Button/Button.tsx

```diff
@@ -164,6 +164,11 @@ export interface ButtonState {
   focusedByTab: boolean;
 }
 
+export const buttonDataTid = {
+  origin: 'Button',
+  root: 'Button__root',
+};
```

Comment:
Я бы предложил называть эти объекты с большой буквы, как и все, что экспортируется из библиотеки, для большей эстетичности. И добавить `s` в конце - это же набор тидов, их там много. Также, указать им `as const`, чтобы сузить тип полей со `string` до конкретных значений, которые пользователь сможет увидеть уже в типах. Плюс, это сделает их `readonly`.

```suggestion
export const ButtonDataTids = {
  origin: 'Button',
  root: 'Button__root',
} as const;
```